src/assets/styles/30-components/__manifest.scss

```diff
@@ -13,3 +13,5 @@
 @import './src/components/Nav/nav-list';
 @import './src/components/SearchPane/search-pane';
 @import './src/components/WellcomeCollectionBanner/wellcome-collection-banner';
+@import './src/Components/Logo/Logo';
+@import './src/components/ResultItem/ResultItem';
```

Comment:
path needs updating due to filename change

Reply:
done.